src/components/pages/Calculator.js

```diff
@@ -0,0 +1,72 @@
+import './calculator.css';
+
+const Calculator = () => (
+  <section className="my-calculator">
+    <div className="result">
+      <span className="screen">
+        0
+      </span>
+    </div>
+    <div className="buttons">
+      <button type="button" className="ac">
+        AC
+      </button>
+      <button type="button" className="plmi">
+        +/-
+      </button>
+      <button type="button" className="percent">
+        %
+      </button>
+      <button type="button" className="division orange">
+        /
+      </button>
```

Comment:
- [ ] This symbol should be &div, not /, based on the design. If you do not have the sign on your keyboard consider using HTML entity code for it.